src/test/java/api/requests/scenarios/MultipleMixedFulfilmentRequestsTests.java

```diff
@@ -26,51 +26,12 @@
 import api.support.APITests;
 
 public class MultipleMixedFulfilmentRequestsTests extends APITests {
-  @Test
-  public void deliveryRequestIsIgnoredWhenItemCheckedIn()
```

Comment:
I think this should be replaced with tests with a mixed queue, demonstrating that the precedence of the queue is respected for both fulfilment preferences.

For example, two tests: one with delivery first, then hold shelf, and vice versa.

Reply:
Thanks, added a unit test and renamed another test, now there are two tests:
deliveryRequestIsProcessedWhenItIsNextInQueueAndItemCheckedIn()
holdShelfRequestIsProcessedWhenItIsNextInQueueAndItemCheckedIn()